statistics/handle/handle_hist.go

```diff
@@ -353,24 +384,29 @@ func (h *Handle) updateCachedColumn(col model.TableColumnID, colHist *statistics
 func (h *Handle) setWorking(col model.TableColumnID, resultCh chan model.TableColumnID) bool {
 	h.StatsLoad.Lock()
 	defer h.StatsLoad.Unlock()
-	chList, ok := h.StatsLoad.workingColMap[col]
+	chList, ok := h.StatsLoad.WorkingColMap[col]
 	if ok {
-		h.StatsLoad.workingColMap[col] = append(chList, resultCh)
+		if chList[0] == resultCh {
+			return true
+		}
```

Comment:
Why we added this checking?

Reply:
The worker taking the loading work is at the head of the list, should not append duplicatedly. If append duplicatedly, there will be duplicated messages in resultCh.

Reply:
Add a comment here.